styles.css

```diff
@@ -1,8 +1,52 @@
 h1 {
   color: black;
+  text-align: center;
 }
```

Comment:
- [ ] Please make sure that your page looks like the provided wireframe, currently there are some mismatches: 

Your layout  | Wireframe layout
------------- | -------------
<img width="1433" alt="Screenshot 2022-03-15 at 20 04 36" src="https://user-images.githubusercontent.com/77038610/158420536-3fa2ceb1-1644-43ea-aed9-4fb984db7326.png">  | <img width="1217" alt="Screenshot 2022-03-15 at 20 04 20" src="https://user-images.githubusercontent.com/77038610/158420474-1313801e-bad3-4855-b3cd-9fa10ea5ff14.png">
More specifically:

- Please align add button to the right
- Please make list of books and its container smaller so that it looked more harmonised on the page with other items
- Please make the form inputs wider 
- Please add a horizontal line between the list of books and form